doc/neps/nep-0029-deprecation_policy.rst

```diff
@@ -111,7 +111,9 @@ Jun 23, 2020 3.7+   1.15+
 Jul 23, 2020 3.7+   1.16+
 Jan 13, 2021 3.7+   1.17+
 Jul 26, 2021 3.7+   1.18+
-Dec 26, 2021 3.8+   1.18+
+Dec 22, 2021 3.7+   1.19+
+Dec 26, 2021 3.8+   1.19+
+Jun 21, 2022 3.8+   1.18+
```

Comment:
This and the next line should be `1.20+` right?

Reply:
Hum, there is likely a bug in the Python at the bottom of the NEP;  I blindly copy pasted. Will look into it:

<img width="386" alt="Screen Shot 2020-09-17 at 08 57 04" src="https://user-images.githubusercontent.com/335567/93496210-cd021b80-f8c3-11ea-9be5-372bf4606352.png">